07_cmd_lab/cmdlab.txt

```diff
@@ -3,11 +3,14 @@ Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+  echo $0
   - your default shell?
+  bash
   - a list of all shells currently installed on the machine?
-
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+  cat /etc/shells
+2. View the file /etc/passwd manon your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
```

Comment:
your'e missing an answer here :P 
(always feel free to pen an issue of contact me directly if you stuck)